src/app/config/config.model.ts

```diff
@@ -67,15 +68,22 @@ interface EnclosurePlugin extends Plugin {
 }
 
 interface PSUControlPlugin extends Plugin {
+  // TODO: this option still exists to allow migration path... need to be removed
+  // when the new `turnOnPSUWhenExitingSleep` will be released
   turnOnPSUWhenExitingSleep: boolean;
```

Comment:
```suggestion
  turnOnPSUWhenExitingSleep?: boolean;
```

we should mark it optional though when it should get removed in the next release :)